WordPress/Classes/Categories/NSDate+StringFormatting.m

```diff
@@ -46,4 +47,362 @@ - (NSString *)shortString
+                return  NSLocalizedString(@"In 12 days", @"In 12 days");
+            } else {
+                return  NSLocalizedString(@"12 days ago", @"12 days ago");
+            }
+        case 11:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 11 days", @"In 11 days");
+            } else {
+                return  NSLocalizedString(@"11 days ago", @"11 days ago");
+            }
+        case 10:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 10 days", @"In 10 days");
+            } else {
+                return  NSLocalizedString(@"10 days ago", @"10 days ago");
+            }
+        case 9:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 9 days", @"In 9 days");
+            } else {
+                return  NSLocalizedString(@"9 days ago", @"9 days ago");
+            }
+        case 8:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 8 days", @"In 8 days");
+            } else {
+                return  NSLocalizedString(@"8 days ago", @"8 days ago");
+            }
+        case 7:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 7 days", @"In 7 days");
+            } else {
+                return  NSLocalizedString(@"7 days ago", @"7 days ago");
+            }
+        case 6:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 6 days", @"In 6 days");
+            } else {
+                return  NSLocalizedString(@"6 days ago", @"6 days ago");
+            }
+        case 5:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 5 days", @"In 5 days");
+            } else {
+                return  NSLocalizedString(@"5 days ago", @"5 days ago");
+            }
+        case 4:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 4 days", @"In 4 days");
+            } else {
+                return  NSLocalizedString(@"4 days ago", @"4 days ago");
+            }
+        case 3:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 3 days", @"In 3 days");
+            } else {
+                return  NSLocalizedString(@"3 days ago", @"3 days ago");
+            }
+        case 2:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"In 2 days", @"In 2 days");
+            } else {
+                return  NSLocalizedString(@"2 days ago", @"2 days ago");
+            }
+        case 1:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"Tomorrow", @"Tomorrow");
+            } else {
+                return  NSLocalizedString(@"Yesterday", @"Yesterday");
+            }
+        case 0:
+            if (isFutureDate) {
+                return  NSLocalizedString(@"Later today", @"Later today");
+            } else {
+                return  NSLocalizedString(@"Today", @"Today");
+            }
```

Comment:
The Reader could also benefit if this new date formatters get in.

On Sat, May 16, 2015 at 8:34 AM, Sergio Estevao <sergio.estevao@a8c.com>
wrote:

> Don't you want to use it on the Post lists also? instead of 170d?
>
> On Sat, May 16, 2015 at 2:46 AM, Eric <notifications@github.com> wrote:
>
>> In WordPress/Classes/Categories/NSDate+StringFormatting.m
>> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3657#discussion_r30424910>
>> :
>>
>> > +                return  NSLocalizedString(@"In 2 days", @"In 2 days");
>> > +            } else {
>> > +                return  NSLocalizedString(@"2 days ago", @"2 days ago");
>> > +            }
>> > +        case 1:
>> > +            if (isFutureDate) {
>> > +                return  NSLocalizedString(@"Tomorrow", @"Tomorrow");
>> > +            } else {
>> > +                return  NSLocalizedString(@"Yesterday", @"Yesterday");
>> > +            }
>> > +        case 0:
>> > +            if (isFutureDate) {
>> > +                return  NSLocalizedString(@"Later today", @"Later today");
>> > +            } else {
>> > +                return  NSLocalizedString(@"Today", @"Today");
>> > +            }
>>
>>  TTTTimeIntervalFormatter
>>
>> I'll take a look. Thanks for the tip!
>>
>> this change (changing the format of Dates intervals) should de done in a
>> separate PR. It's not related to the Pages redesign itself
>>
>> This formatting *is* specific to the pages list and not used elsewhere.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3657/files#r30424910>
>> .
>>
>
>